Marsiple/App/Tab Bar/TabBarController.swift

```diff
@@ -13,6 +13,9 @@ class TabBarController: UITabBarController {
         super.init(nibName: nil, bundle: nil)
         let navigationController = UINavigationController(rootViewController: PostsViewController())
         navigationController.navigationBar.prefersLargeTitles = true
+        navigationController.navigationBar.barTintColor = MatrianColor.gray.value
```

Comment:
Wouldn't it be cool if we could use custom colors like-so:
`navigationController.navigationBar.barTintColor = .martianGray`

Do you have an idea how you would accomplish this?

Reply:
We could do this in ÙIColor`extension using computed properties.